src/resources/js/components/Presentation.vue

```diff
@@ -93,6 +99,24 @@ export default {
           alert.showMessage('error', 'Oops une erreur c\'est produite lors de la désinscription');
         });
     },
+    isPresenter() {
+      return (this.presentation.auth_user_role === 'presenter');
+    },
+    delPresentation(id) {
+      const apiUrl = `/presentations/${id}`;
+
+      window.axios.delete(apiUrl)
+        .then(() => {
+          // go back to presentation list
+          alert.showMessage('success', 'La presentation à été supprimé correctement');
+          setTimeout(() => {
+            this.$router.push('/presentations');
+          }, 2000);
```

Comment:
remove element from the dom instead of reloading the page to keep a nice user experience

Reply:
ok, it will hide the elem when deleted button is trigger [ade9f](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/161/commits/ade9f70203eca1b72ca0996e13c9944ea7bc48fe)